src/index_intf.ml

```diff
@@ -169,6 +172,10 @@ module type S = sig
   (** [is_merging t] returns true if [t] is running a merge. Raises
       {!RO_not_allowed} if called by a read-only index. *)
 
+  val merge : t -> unit
+  (** [merge t] forces a merge for [t]. It is non-blocking, i.e. it returns
+      immediately, with the merge running concurrently. *)
```

Comment:
It does block if a merge is already running, right?

Reply:
Indeed. I changed it, let me know if it's clear